udata/templates/reuse/display.html

```diff
@@ -55,8 +55,13 @@
 {% endblock %}
 
 {% block content %}
+<div itemscope="itemscope" itemtype="http://schema.org/Dataset" itemid="{{ reuse.id }}">
```

Comment:
I don't think Dataset fits.
Maybe [CreativeWork](https://schema.org/CreativeWork) or one of its children, depending on the reuse type.